tree/ntuple/v7/src/RNTupleSerialize.cxx

```diff
@@ -124,51 +134,62 @@ ROOT::RResult<std::uint32_t> DeserializeField(const void *buffer, std::uint64_t
    auto bytes = base;
    std::uint64_t frameSize;
    auto fnFrameSizeLeft = [&]() { return frameSize - (bytes - base); };
-   auto result = RNTupleSerializer::DeserializeFrameHeader(bytes, bufSize, frameSize);
-   if (!result)
-      return R__FORWARD_ERROR(result);
-   bytes += result.Unwrap();
+   if (auto res = RNTupleSerializer::DeserializeFrameHeader(bytes, bufSize, frameSize)) {
+      bytes += res.Unwrap();
+   } else {
+      return R__FORWARD_ERROR(res);
+   }
```

Comment:
This part (and similar below) seem to be non-functional changes, is that correct?  If so, what is the advantages of the new formulation?

Reply:
The advantage is to reduce the scope of the `res`/`result` variable to only the `if` statement.

Reply:
The actual reason is that there were like 50 usages of `UNWRAP_OR_RETURN` and I didn't check case-by-case. I hardly believe this kind of minor syntactical details are worth the time we are spending on them, as in the end it's all equivalent.

Reply:
Is this a blocker? Or can this conversation be resolved?

Reply:
No, let's do it uniformly everywhere, resolving from my side.